BetterPasswordPlugin.inc.php

```diff
@@ -208,7 +198,94 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+	/**
+	 * Check for newly added blacklists. If ran the first time creates the list of blacklists
+	 * @return boolean false if update failed, true if updated the setting
```

Comment:
This is actually returning true if the blacklist is usable, not if the setting is updated.